src/test/java/core/basesyntax/service/FruitShopServiceTest.java

```diff
@@ -0,0 +1,75 @@
+package core.basesyntax.service;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitModel;
+import core.basesyntax.service.impl.FruitShopServiceImpl;
+import core.basesyntax.strategy.handler.OperationHandler;
+import core.basesyntax.strategy.handler.impl.BalanceOperationHandler;
+import core.basesyntax.strategy.handler.impl.OperationStrategyImpl;
+import core.basesyntax.strategy.handler.impl.PurchaseOperationHandler;
+import core.basesyntax.strategy.handler.impl.ReturnOperationHandler;
+import core.basesyntax.strategy.handler.impl.SupplyOperationHandler;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class FruitShopServiceTest {
+    private static final String BANANA = "banana";
+    private static final String APPLE = "apple";
+    private static final Integer BANANA_QUANTITY = 152;
+    private static final Integer APPLE_QUANTITY = 90;
+    private static FruitShopService fruitShopService;
+    private static List<FruitModel> fruitModels;
+
+    @BeforeClass
+    public static void beforeAll() {
+        Map<FruitModel.Operation, OperationHandler> operationHandlerMap = Map.of(
+                FruitModel.Operation.BALANCE, new BalanceOperationHandler(),
+                FruitModel.Operation.PURCHASE, new PurchaseOperationHandler(),
+                FruitModel.Operation.SUPPLY, new SupplyOperationHandler(),
+                FruitModel.Operation.RETURN, new ReturnOperationHandler()
```

Comment:
you can use static imports instead of referencing FruitModel.Operation.<Operation>